airflow/models/connection.py

```diff
@@ -28,6 +28,50 @@
 from airflow.models.base import ID_LEN, Base
 from airflow.models.crypto import get_fernet
 from airflow.utils.log.logging_mixin import LoggingMixin
+from airflow.utils.module_loading import import_string
+
+# A map that assigns a connection type to a tuple that contains
+# the path of the class and the name of the conn_id key parameter.
+# PLEASE KEEP BELOW LIST IN ALPHABETICAL ORDER.
```

Comment:
@mik-laj - maybe add a test for that. We already have quite a few tests where we check if we have alphabetical order in source code.

Reply:
It should be easy as as of python 3.6 all the dicts are ordered by insertion order https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6  so our tests should just test is keys are sorted.

Reply:
Good idea. I will add it.